hlxsites/merative2
# fix(blog-detail): add correct spacing to paragraph

## Issue

Fixes https://github.com/hlxsites/merative2/issues/246

## Description

**Changed**

- This PR adds the correcting spacing between paragraphs for desktop and smaller breakpoints.
- Currently, we have `16px` spacing and the text paragraphs are too close (see - https://www.merative.com/blog/marcy-tatsch-curam).
- Adds styling for the unordered list which was not part of the Blog Detail page Figma design specs.


## Design Specs


- Figma Link - https://www.figma.com/file/FSqQ1LYrVSssEFOZuOnMIE/Final-page-layouts-for-Merative.com?type=design&node-id=1262%3A48026&t=jPkVX4hrssQJtxqi-1

## Test URLs
  
![Screen Shot 2023-06-20 at 3 02 36 PM](https://github.com/hlxsites/merative2/assets/1815714/2585386b-0a43-4058-b1e2-95124bdbb985)

![Screen Shot 2023-06-20 at 4 37 58 PM](https://github.com/hlxsites/merative2/assets/1815714/3fb83f9f-d9f1-44b7-a62e-180097b9a62d)

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.live/blog/marcy-tatsch-curam
- After (Changes from this PR): https://fix-blog-paragraph-spacing--merative2--proeung.hlx.page/blog/marcy-tatsch-curam
  
## Testing Instruction

- Visit one of the blog pages (eg. https://fix-blog-paragraph-spacing--merative2--proeung.hlx.page/blog/ism-recap-2022)
- Ensure that the spacing on the desktop and smaller breakpoints match with the Figma design 
- Ensure that there's proper styling of the `<ol>` or unordered list
